src/GlobalStyles/global.style.ts

```diff
@@ -0,0 +1,28 @@
+import { createGlobalStyle, css } from "styled-components";
+import { Colors } from "..";
```

Comment:
Update to the full import path as discussed :) 

like sidebar:
`import Colors from "../Colors";`